crypto/x509/x_all.c

```diff
@@ -19,10 +19,122 @@
 #include <openssl/dsa.h>
 #include <openssl/x509v3.h>
 
+#ifndef OPENSSL_NO_SM2
+
+# include "internal/asn1_int.h"
+# include "internal/evp_int.h"
+
+static int x509_verify_sm2(X509 *x, EVP_PKEY *pkey, int mdnid, int pknid);
```

Comment:
Why this line? The actual function is just below it. Delete.

Reply:
Okay

Reply:
Removed in the latest commit